internals/webpack/webpack.base.babel.js

```diff
@@ -7,22 +7,25 @@ const webpack = require('webpack');
 
 module.exports = (options) => ({
   entry: options.entry,
-  output: { // Compile into js/build.js
-    path: path.resolve(__dirname, '../..', 'build'),
-    filename: '[name].js',
-    chunkFilename: '[name].chunk.js',
+  output: Object.assign({ // Compile into js/build.js
+    path: path.resolve(__dirname, '..', 'build'),
     publicPath: '/',
-  },
+  }, options.output), // Merge with env dependent settings
   module: {
     loaders: [{
       test: /\.js$/, // Transform all .js files required somewhere with Babel
       loader: 'babel',
-      exclude: path.join(__dirname, '../..', '/node_modules/'),
+      exclude: /node_modules/,
       query: options.babelQuery,
     }, {
       test: /\.css$/, // Transform all .css files required somewhere with PostCSS
+      exclude: /node_modules/,
       loader: options.cssLoaders,
     }, {
+      test: /\.css$/,
```

Comment:
Can we add a comment here and change the above comment to explain why we have two different loaders for `.css` files?